games/nethack/Portfile

```diff
@@ -7,7 +7,9 @@ version            3.6.1
 categories         games
 platforms          darwin
 license            Copyleft
-maintainers        @Nax
+maintainers        @Nax \
+                   {@jflude hotmail.com:justin_flude} \
+                   openmaintainer
```

Comment:
Cleaner and less redundant logic for handling the preservation of game state in the main and subports.